src/components/Button/Button.tsx

```diff
@@ -34,7 +34,7 @@ export interface IButtonProps {
   onFocus?: ComponentEventHandler<IButtonProps>
   renderIcon?: ShorthandRenderFunction
   text?: boolean
-  type?: 'primary' | 'secondary'
+  type?: 'primary'
```

Comment:
do we really want to remove for all the Stardust clients? What is the point to have `type` being a string with `secondary` removed - it should be a `boolean` flag then. However, would disagree that we should remove `secondary` from the list of types provided.

Reply:
What does "secondary" mean though, vs just a "default" button? Or conversely, if we have a "primary" and "secondary" button visual treatment, then the "neither primary nor secondary button" becomes a liability (in the context of Teams at least) because there is no such thing as a button that isn't primary or secondary... do we make "primary" and "secondary" non-optional for button? That seems onerous.

I agree that perhaps "primary" should be a Boolean rather than a string... This might also be useful in "groups" of buttons, to make some sort of rule that breaks if there is more than one primary...